stable/firmament-poseidon/templates/rbac.yaml

```diff
@@ -0,0 +1,131 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+    app: {{ template "firmament-poseidon.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Values.clusterrolebinding.name }}
```

Comment:
Resource names should be based on the fullname parial. Is there a reason why you want this to be configurable?